react-slider.js

```diff
@@ -749,6 +752,10 @@
             "aria-valuenow": this.state.value[i],
             "aria-valuemin": this.props.min,
             "aria-valuemax": this.props.max,
+            "aria-label": ariaLabelArray && i < this.props.ariaLabel.length
```

Comment:
Bound check is not necessary here (in fact it's an error because when out of bound access is detected it will pass an array to `aria-label` property). We can simply write it as follows.

```javascript
"aria-label": isArray(this.props.ariaLabel) ? this.props.ariaLabel[i] : this.props.ariaLabel,
```

Reply:
That's right! Changing this.